gapis/gfxapi/sync/sync.go

```diff
@@ -41,6 +41,9 @@ type SynchronizedAPI interface {
 	// ResolveSynchronization resolve all of the synchronization information for
 	// the given API
 	ResolveSynchronization(ctx context.Context, d *Data, c *path.Capture) error
+
+	// MutateSubcommands mutates the given Atom calling callback after each subcommand is executed.
+	MutateSubcommands(ctx context.Context, a atom.Atom, id atom.ID, s *gfxapi.State, callback func(*gfxapi.State, []uint64, atom.Atom)) error
```

Comment:
Should we change the `[]uint64` to `SubcommandIndex` to make the interface clearer?